homeassistant/components/flunearyou/sensor.py

```diff
@@ -102,23 +131,20 @@ def __init__(
         self,
         coordinator: DataUpdateCoordinator,
         entry: ConfigEntry,
-        sensor_type: str,
-        name: str,
-        icon: str,
-        unit: str | None,
+        description: SensorEntityDescription,
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
+        self._sensor_type = description.key
```

Comment:
Replace the use of `self._sensor_type` with `self.entity_description.key` in the code, so this variable can be removed.